packages/core/content-type-builder/server/src/controllers/component-categories.ts

```diff
@@ -18,7 +18,7 @@ export default {
 
     const componentCategoryService = getService('component-categories');
 
-    const newName = await componentCategoryService.editCategory(name, body);
+    const newName = await componentCategoryService.editCategory(name, body as any);
```

Comment:
All of these seemingly unrelated body typing changes are a result of typescript build errors I started getting, but it appears they are valid; body is unknown so we shouldn't be passing it like this without checking. But that's out of scope for this PR, so I'm just making it build.

Reply:
Confirmed they happen in v5/main for me too right now. I'll wait until that's solved before merging, but in general this PR is still ready to review.